.github/ci/packages-jammy.apt

```diff
@@ -0,0 +1 @@
+python3-distutils
```

Comment:
I don't think we need it in Jammy either (see https://github.com/gazebosim/gz-sim/pull/2374#discussion_r1572629902)

Reply:
you're right; I knew we need it for focal and wasn't thinking clearly about jammy. removed in https://github.com/gazebosim/gz-math/pull/590/commits/355780f8dd2fa40ec3d16c377b655406f84d1181